src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
 ---
-title: "Tools to measure Core Web Vitals"
-subhead: Your favorite developer tools can now measure the Core Web Vitals.
+layout: post
+title: Core Web Vitals workflows with Google tools
+subhead: Combine Google tools to audit, improve and monitor your website effectively.
 authors:
+  - antoinebisch
+  - gmimani
   - addyosmani
   - egsweeny
-date: 2020-05-28
-description: |
-  Read about the newly announced Core Web Vitals measurement support across popular web developer tools like Lighthouse, PageSpeed Insights, Chrome UX Report, and many others.
-hero: image/admin/wNtXgv1OE2OETdiSzi8l.png
-thumbnail: image/admin/KxBRBQe5CRZpCxNYyW2H.png
-alt: Chrome User Experience logo, PageSpeed Insights logo, Lighthouse logo, Search Console logo, Chrome DevTools logo, Web Vitals extension logo.
+date: 2022-01-14
+hero: image/PTlEGTxNW2XkfI8mA3pIGDFxdYY2/1qMzdR1XGj3HKzuW4PFG.jpg
+thumbnail: image/PTlEGTxNW2XkfI8mA3pIGDFxdYY2/ApVe49OiB4KIJwcC7njy.jpg
+alt: An image of icons of the various Google tools described in this article.
+description: >
+  With the growing importance of Core Web Vitals, site owners and developers increasingly focus on performance and key user experiences. Google provides many tools to help evaluate, optimize, and monitor pages, but users are often confused by the different sources of data and how to leverage them effectively. This article proposes a workflow combining several tools and clarifies where and how they make sense along the development process.
 tags:
-  - blog # blog is a required tag for the article to show up in the blog.
+  - blog
   - web-vitals
   - performance
 ---
 
-The recently announced [Web Vitals](/vitals/) initiative provides unified guidance about quality signals that are essential for all sites to deliver a great user experience on the web. We're happy to announce that **all of Google's popular tools for web developers now support measurement of Core Web Vitals**, helping you more easily diagnose and fix user experience issues. This includes [Lighthouse](https://github.com/GoogleChrome/lighthouse), [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/), [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools), [Search Console](https://search.google.com/search-console/about), [web.dev's measure tool](/measure/), the [Web Vitals Chrome extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma) and a new (!) [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report) API.
+[Core Web Vitals](/vitals/) are a set of metrics that assess the user experience on criteria such as perceptual performance, responsiveness to user input, and layout stability. While Core Web Vitals are valuable user experience metrics, they won't mean much without guidance on creating a development workflow to continuously improve them in both the lab and the field.
```

Comment:
I'd replace the term "perceptual performance" with "load performance". While _I_ understand what you mean when you say this, [we don't use this term anywhere else on web.dev](https://www.google.com/search?q=%22perceptual+performance%22+site%3Aweb.dev&oq=%22perceptual+performance%22+site%3Aweb.dev), so I think it's better not to introduce it here.